lib/rules/rule-selector-property-disallowed-list/index.mjs

```diff
@@ -71,3 +75,26 @@ rule.ruleName = ruleName;
 rule.messages = messages;
 rule.meta = meta;
 export default rule;
+
+/**
+ * Check that a given declaration is a child of this rule and not a nested rule
+ *
+ * @param {import('postcss').Declaration} decl
+ * @param {import('postcss').Rule} ruleNode
+ */
+function declarationIsAChildOfRule(decl, ruleNode) {
+	if (decl.parent === ruleNode) return true;
```

Comment:
[suggest] This code seems to be the same as `if (parent === ruleNode) return true;` in the `while` block, so maybe we may remove it:

```suggestion
```